apps-rendering/cdk/lib/mobile-apps-rendering.test.ts

```diff
@@ -0,0 +1,24 @@
+import '@aws-cdk/assert/jest';
+import { SynthUtils } from '@aws-cdk/assert';
+import { App } from '@aws-cdk/core';
+import { MobileAppsRendering } from './mobile-apps-rendering';
+
+describe('The MobileAppsRendering stack', () => {
+	it('matches the snapshot', () => {
```

Comment:
Has the formatting gone wonky here? Looks like there is loads more whitespace in this one?

Reply:
The indentation issue was fixed.